react/react1/week2/react-week2/src/components/TodoApp.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import {useState } from "react";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faTrash } from '@fortawesome/free-solid-svg-icons'
+// import { seconds } from "./TodoTimer";
+
+const todos = [
+{id: 1, description: "Get out of bed", deadline: "Wed Sep 13 2017" }, 
+{id: 2, description: "Brush teeth", deadline: "Thu Sep 14 2017"}, 
+{id: 3, description: "Eat breakfast", deadline: "Fri Sep 15 2017" },
+{id: 4, description: "Eat launch", deadline: "Fri Sep 16 2017" }
+];
```

Comment:
In React we try to separate our concerns as much as we can, so it would make it easier to manage our app, in that regard, it's better to have your data separated in another file, which a name suggestion for it is `todosCollection.js`

Reply:
ok